### DISCUSSION TITLE
React-Native: Appropriate Loader error (import typeof AccessibilityInfo)

### DISCUSSION THREAD

**Discussion by benjgil** [teambit/bit#5865] [FAQ]
If you get the following error when trying to build your react-native components, then it's almost certainly due to the component not using a react-native env.

Run `bit env` and check that the env listed for the component in question is in fact a react-native env.
If it's not configured with the env you thought, run `bit env set <component-id> <env-full-id>` to set the required env.

Screenshot of the error:
![Screenshot from 2022-03-30 14-17-27](https://user-images.githubusercontent.com/67269267/170467165-67fe9c6c-6608-43b4-bfcd-7104e30375cf.png)

**Comment by Saikedo** [comment]
@benjgil Any idea how this can be addressed when importing a 3rd party library ?

I am trying to include a 3rd party library that works perfectly fine in react native web environment but gives `You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders` error when trying to preview.

I tried solving this by configuring the webpack to use babel-loader and include node_modules/someComponent but was unable to fix the issue.
